.travis.yml

```diff
@@ -3,30 +3,46 @@ services:
 - docker
 install:
 - pip install twine wheel pyOpenSSL
-script:
-- |
+- pip install -r requirements.txt
 
-  export last_commit_author=`git log -1 --pretty=format:%ae`
+branches:
+  only:
+  - /^feature.*$/
+  - master
 
-  docker build -t finbourne/lusid-python-tools-test .
+jobs:
+  include:
+    - stage: Run black formatter and commit any changes
+      if: branch != master
+      script:
+        - bash ci_tools/new_commit_with_black.sh
+    - stage: Run LPT package tests 
+      script:
+      - |
 
-  docker run \
-      -e FBN_TOKEN_URL="$fbn_token_url" \
-      -e FBN_USERNAME="$fbn_username" \
-      -e FBN_PASSWORD="$fbn_password" \
-      -e FBN_CLIENT_ID="$fbn_clientid" \
-      -e FBN_CLIENT_SECRET="$fbn_client_secret" \
-      -e FBN_LUSID_API_URL="$fbn_lusid_api_url" \
-      -e FBN_APP_NAME="$fbn_app_name" \
-      -v $(pwd):/usr/src/ \
-      finbourne/lusid-python-tools-test
+        export last_commit_author=`git log -1 --pretty=format:%ae`
```

Comment:
I don't think you need this line any more

Reply:
Agreed. Removed.